nixos/doc/manual/from_md/release-notes/rl-2111.section.xml

```diff
@@ -360,6 +360,19 @@
           <literal>rxvt-unicode</literal> explicitly.
         </para>
       </listitem>
+      <listitem>
+        <para>
+          The linux kernel package infrastructure was moved out of
+          <literal>all-packages.nix</literal>, and restructured. Linux
+          related functions and attributes now live under the
+          <literal>pkgs.linuxKernel</literal> attribute set. The
+          versioned <literal>linuxPackages_*</literal> package sets and
+          kernels from <literal>pkgs</literal> are now considered
+          deprecated, and kept for backwards compatibility only. If you
+          want to pin the kernel package to a specific version, use one
+          of the <literal>linuxKernel.packages.*</literal> attributes.
```

Comment:
So they weren't deprecated after all. The suspense was killing me. Perhaps you could describe it is a move rather than a deprecation?

I don't think we need to document the existence of the compatibility aliases here. People will find out easily enough that they're deprecated because of the warnings.